class.jetpack.php

```diff
@@ -6932,22 +6933,23 @@ public static function build_raw_urls( $url ) {
 	 */
 	public static function dns_prefetch( $new_urls = null ) {
```

Comment:
Since the `wp_resource_hints` hook accepts 2 parameters, you would have to specify `$relation_type` here, as well as set a default value for it.